charts/kubernetes-agent/README.md.gotmpl

```diff
@@ -1,13 +1,16 @@
 {{ template "chart.header" . }}
 
-{{ template "chart.badgesSection" . }}
+{{ template "chart.badgesSection" . }}![Octopus Deploy Version: 2024.2.6580+](https://img.shields.io/badge/Octopus_Deploy-2024.2.6580%2B-2F93E0?style=flat-square&logo=octopusdeploy&logoColor=%232F93E0&logoSize=auto)
 
 {{ template "chart.description" . }}
 
-{{ template "chart.homepageLine" . }}
+{{ template "chart.homepageLine" . }}  
+**Documentation:** [https://octopus.com/docs/](https://octopus.com/docs/infrastructure/deployment-targets/kubernetes/kubernetes-agent)
 
 {{ template "chart.maintainersSection" . }}
 
 {{ template "chart.sourcesSection" . }}
 
-{{ template "chart.valuesSection" . }}
\ No newline at end of file
+{{ template "chart.valuesSection" . }}
+
+{{ template "helm-docs.versionFooter" . }}
```

Comment:
Is there a reason we're adding the `helm-docs` footer in?

Reply:
Yeah. I think it's appropriate that we acknowledge the use of this tool via the footer. Give readers information that they might want to use the tool as well

Reply:
The only reason I didn't was to stop the version from showing diffs when the container updates on a dev machine vs. CI, not to hide the fact we use the tool 😅

Reply:
oo, fair... I'll remove it this and replace with with a fixed footer that won't change